scenarios/aws/ec2/vm.go

```diff
@@ -64,7 +64,7 @@ func NewVM(e aws.Environment, name string, params ...VMOption) (*remote.Host, er
 		if vmArgs.osInfo.Family() == os.WindowsFamily {
 			randomPassword, err := random.NewRandomString(e.Ctx(), e.Namer.ResourceName(name, "win-admin-password"), &random.RandomStringArgs{
 				Length:  pulumi.Int(20),
-				Special: pulumi.Bool(true),
+				Special: pulumi.Bool(false),
```

Comment:
You could use [OverrideSpecial](https://www.pulumi.com/registry/packages/random/api-docs/randomstring/#overridespecial_go) if the goal is just not to have dollar signs, but I guess removing special characters entirely works too !

Reply:
Yeah I do not have the full list of invalid characters, but `{ }` are an issue as well. So it is safer to just not put any special character